scripts/enrich_rulings_ai.py

```
@@ -0,0 +1,276 @@
+            text_for_ai = enriched_ruling['text']
+        elif enriched_ruling.get("original_html_snippet"):
+            # For already processed ArkhamDB rulings, original_html_snippet is HTML.
+            # For external rulings, it's raw text.
+            # We need plain text for AI.
+            soup = BeautifulSoup(enriched_ruling["original_html_snippet"], "html.parser")
+            text_for_ai = soup.get_text(separator=' ', strip=True)
+
+        if not text_for_ai and enriched_ruling.get("original_html_snippet"): # Fallback for external that might be plain
+             text_for_ai = enriched_ruling["original_html_snippet"]
+
+
```

Ruff (0.11.9)

Line too long (121 > 100)

(E501)